src/forms/medication-dispense-review.component.tsx

```diff
@@ -506,13 +501,14 @@ const MedicationDispenseReview: React.FC<MedicationDispenseReviewProps> = ({
           labelText={t('dispenseDate', 'Date of Dispense')}
           minDate={prescriptionDate ? dayjs(prescriptionDate).startOf('day').toDate() : null}
           maxDate={dayjs().toDate()}
-          onChange={(selectedDate) => {
-            const currentDate = dayjs(medicationDispense.whenHandedOver);
+          onChange={(input) => {
```

Comment:
the OpenMRS datepicker now returns a Date, not a CalenderDate, which makes things simpler